### PR TITLE
Joe work

### PR DESCRIPTION
**Pull request by almet**
Use absolute URLs whenever possible, and also fix a webpack problem.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "A tool to build forms on the web",
   "scripts": {
-    "build:formbuilder": "NODE_ENV=production rimraf build && webpack --config webpack.config.prod.js --optimize-minimize && cp formbuilder/index.prod.html build/index.html",
```
Comment (n1k0) [inline package.json]: Hmm no more minification and dead code removal?

**Comment by n1k0** [comment]
LGTM, just e sure that a non-minified/optimized bundle is what you really want.

**Comment by almet** [comment]
Good catch, updated.

**Pull request merged**